overlay.d/05rhcos/usr/lib/systemd/system-generators/bootloader-migrate-generator

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
Maybe consider sourcing generator-lib.sh here like we [do in our other generators](https://github.com/coreos/fedora-coreos-config/blob/13020cbf38f481d0cd5f7cefb9bb7ea9c627e133/overlay.d/05core/usr/lib/systemd/system-generators/coreos-boot-mount-generator#L5) to get [logging](https://github.com/coreos/fedora-coreos-config/blob/13020cbf38f481d0cd5f7cefb9bb7ea9c627e133/overlay.d/05core/usr/lib/coreos/generator-lib.sh#L3-L5).

cc @jlebon in case that is a bad idea.